packages/ui-library/src/elements/autocomplete/style.css

```diff
@@ -55,7 +63,7 @@
 		.yst-autocomplete__options {
 			@apply
 			yst-absolute
-			yst-z-10
+			yst-z-20
```

Comment:
Yes this component is only used here and in the user picker, as well in shopifiy. and there is no change in existing UI :) Since this was only needed when this component is all the way at the bottom of the screen.